app/models/assignments_prerequisite_assignment.rb

```diff
@@ -1,4 +1,5 @@
 class AssignmentsPrerequisiteAssignment < ApplicationRecord
+  self.primary_key = :assignment_id
```

Comment:
I think the table missing the primary is a consequence of using `create_join_table ` in the database migration. Assuming this was added for making the data migration to work, I think this is not required. Have suggested a simplified in the data migration file.

Reply:
Have removed this.